faqs/pt/cdp.md

```diff
@@ -0,0 +1,137 @@
+
+Quais são os riscos associados à posse de um CDP?
+
+Possuir um CDP é inerentemente arriscado. Existem quatro categorias principais a serem consideradas ao usar um CDP: riscos de mercado, riscos de usuários, riscos de sistema e riscos de variabilidade de parâmetros.
+
+Riscos de mercado
+
+O uso de um CDP envolve assumir dívidas e transferir a propriedade de ativos para um contrato inteligente que possa vendê-los em caso de desaceleração do mercado. Todo CDP com dívida tem um Preço de Liquidação, o preço pelo qual o CDP seria liquidado. O uso de um CDP para fazer alavancagem (https://www.investopedia.com/terms/l/leverage.asp) apresenta outra camada de risco. O potencial de recompensa é maior por meio da alavancagem, mas o potencial de perda também é ampliado. Uma prática usual é manter um alto índice de colateralização em preparação para falhas de mercado e, assim, evitar a liquidação.
+
+Riscos do usuário
+
+Esses são os riscos associados aos erros do usuário. O MakerDAO não tem a capacidade de reverter transações ou recuperar fundos enviados para endereços ou contratos incorretos.
+
+Riscos do sistema
+
+Existem muitos riscos em potencial que a plataforma Maker enfrenta. A lista a seguir tenta destacar alguns dos riscos do sistema, mas nem todos são incluídos:
+
+- Um ataque de hackers contra a infraestrutura de contrato inteligente
+- Um evento de 'cisne negro' em um ou vários tipos de garantias
+- Erros de preço, irracionalidade e circunstâncias imprevistas
+- Falha na infraestrutura centralizada. Por exemplo: falha na conexão com a Internet, erros no MetaMask, etc.
+
+Riscos de variabilidade de parâmetros
+
+É importante observar que os proprietários de CDP estão sujeitos a alterações nos Parâmetros de risco que governam o sistema. Isso equivale a riscos financeiros para o proprietário do CDP. A seguir, é apresentada uma lista incompleta dos parâmetros sujeitos a alterações:
+
+- Taxa de estabilidade
+- Índice de Liquidação
+- Limite máximo da dívida do tipo colateral
+- Pena de liquidação
+
+## Quais são as práticas comuns para limitar riscos?
+
+Os riscos de mercado podem ser reduzidos usando alertas de preço, mantendo um alto Índice de Colateralização, monitorando a saúde do CDP e mantendo reservas suficientes fora do CDP para pagar a dívida ou adicionar mais garantias. Algumas pessoas podem querer enfrentar muitos riscos ou fazer apostas delicadas tentando prever o mercado. Isso pode levar à liquidação do seu CDP, resultando em perdas.
+
+Os riscos do usuário podem ser reduzidos através da prática, usando pequenas quantidades de evidências e verificando minuciosamente os endereços com os quais ele interage.
+
+## Quais são alguns dos benefícios de ter um CDP?
+
+** Pagamento flexível **
```

Comment:
The spaces around the text will break the bold markdown. 

Should be `**Pagamento flexível**`